### ISSUE TITLE
Remove references to `tl build` command from documentation

### ISSUE THREAD

**Issue by pbdot** [teal-language/tl#871]
Using the 24.1 exe from Releases and the docs:

```tl build will compile your project via the rules defined in tlconfig.lua.```

Get
```
PS C:\Dev\RFTeal> tl build
Usage: tl [-h] [--global-env-def <dtlfilename>] [-I <directory>]
       [--wdisable <warning>] [--werror <warning>]
       [--feat-arity {off,on}]
       [--gen-compat [{off,optional,required}]]
       [--gen-target {5.1,5.3,5.4}] [--version] [-q] [-p] [<command>]
       ...

Error: unknown command 'build'
```

and 

```
Commands:
   check                 Type-check one or more Teal files.
   gen                   Generate a Lua file for one or more Teal files.
   run                   Run a Teal script.
   warnings              List each kind of warning the compiler can produce.
   types                 Report all types found in one or more Teal files
```

Cyan is required?

**Comment by euclidianAce** [comment]
Yes, the `build` command was removed from `tl` due to its complexity. Cyan is required. Apparently some documentation didn't get updated.

**Comment by pbdot** [comment]
I am liking Teal, though first contact with it is pretty rough, cyan isn't mentioned anywhere on main GitHub page or in the tutorial, and installing Cyan/Teal on Windows through LuaRocks has significant land mines.

**Comment by hishamhm** [comment]
Fixed via [e54b7c7](https://github.com/teal-language/tl/commit/e54b7c72108efd8969d5106803e5c8f51846878c)!

> I am liking Teal, though first contact with it is pretty rough, cyan isn't mentioned anywhere on main GitHub page or in the tutorial, and installing Cyan/Teal on Windows through LuaRocks has significant land mines.

That is good feedback, thank you! It might be a good idea to start distributing a precompiled `cyan.exe` as well. Perhaps [lrc](https://lrocket.codeberg.page/) could help with that!
